src/App.js

```diff
@@ -0,0 +1,23 @@
+import {
+  BrowserRouter, Route, Routes,
+} from 'react-router-dom';
+import './App.css';
+import NavBar from './components/NavBar';
+import BooksPage from './routes/BooksPage';
+import Categories from './routes/Categories';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <NavBar />
+
+      <Routes>
+        <Route path="/" element={<BooksPage />} />
+        <Route path="/Books" element={<BooksPage />} />
```

Comment:
- [ ] Nice job with the react-router configuration, You have added the book page twice with different routes, So please remove the second one `<Route path="/Books" element={<BooksPage />} />`.
If you want to make your routes better you can do it like that:
```javascript
<Routes>
        <Route path="/" element={<BooksPage />} />
        <Route path="Categories" element={<Categories />} />
        <Route path="*" element={<BooksPage />} /> // that will redirect any other routes to the main page(books page)
<Routes>
```

Reply:
Extremely grateful for your code suggestion, you're the best! @Omar-Muhamad